### DISCUSSION TITLE
FR: Automatically set the group membership of users

### DISCUSSION THREAD

**Discussion by cfrietschy** [rustdesk/rustdesk-server-pro#473] [Feature Request]
Please add a function to automatically set the group membership based on values from Azure, so that users newly logged in via Azure are automatically assigned to a group and do not end up in “Default”.

**Comment by CIAIT** [comment]
Perhaps it would be helpful if you could associate an Azure Group with an RD Group. This way you can manage your access based on Azure AD Dynamic groups.
